dask_cuda/initialize.py

```diff
@@ -13,13 +39,16 @@
 
 
 def _create_cuda_context_handler():
-    if int(os.environ.get("DASK_CUDA_TEST_SINGLE_GPU", "0")) != 0:
-        try:
-            numba.cuda.current_context()
-        except numba.cuda.cudadrv.error.CudaSupportError:
-            pass
+    if DASK_USE_ROCM:
```

Comment:
@younseojava IMO you could assume that an AMD GPU is available when someone uses this code.